src-tauri/src/main.rs

```diff
@@ -1,8 +1,17 @@
 // Prevents additional console window on Windows in release, DO NOT REMOVE!!
 #![cfg_attr(not(debug_assertions), windows_subsystem = "windows")]
 
+use tauri::Manager;
+
+mod spito_helper;
+
```

Comment:
```suggestion

#[cfg(target_os = "linux")]
```

We should add it in order to allow compilation only for linux